pkg/interface/src/views/apps/chat/components/profile-overlay.js

```diff
@@ -91,7 +91,7 @@ export class ProfileOverlay extends PureComponent {
 
     const identityHref = group.hidden
       ? '/~profile/identity'
-      : `/~groups/view${association['group-path']}/${window.ship}`;
+      : '/~404';
```

Comment:
Which means if we click on someone's sigil in the overlay we now get a 404? We should just not link to anything in that case.